### PR TITLE
Added a workflow for the Core Weekly

### LINKED ISSUE #60: Generate a Core Weekly Pull Request every Monday by using a GitHub Action

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Added a workflow for the Core Weekly
| Fixed ticket? | Fixes PrestaShop/core-weekly-generator#60

### CODE REVIEW

File: .github/workflows/coreweekly.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Core Weekly
+
+on:
+  schedule:
+    - cron:  '0 7 * * 1'
+#  push:
+#    branches:
+#      - master
```
Comment (PierreRambaud) [inline .github/workflows/coreweekly.yml]:
```suggestion
```

File: .github/workflows/coreweekly.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        path: ./core-weekly-generator
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        cd ./core-weekly-generator
+        pip install -r requirements.txt
+    - name: Generate Core Weekly for week 2
```
Comment (PierreRambaud) [inline .github/workflows/coreweekly.yml]:
```suggestion
    - name: Generate Core Weekly for the current week
```

File: .github/workflows/coreweekly.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        cd ./core-weekly-generator
+        pip install -r requirements.txt
+    - name: Generate Core Weekly for the current week
+      run: |
+        cd ./core-weekly-generator
+        python core-weekly.py --week ${date +"%U"} > ../prestashop.github.io/news/_posts/core-weekly/${date +"%Y-%m-%d-coreweekly-%U-%Y.md"}
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v3
+      with:
+        commit-message: Added Core Weekly
```
Comment (eternoendless) [inline .github/workflows/coreweekly.yml]: Can we add the date here?

File: .github/workflows/coreweekly.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        delete-branch: true
+        title: 'Added Core Weekly'
```
Comment (eternoendless) [inline .github/workflows/coreweekly.yml]: Can we add the date here?

File: .github/workflows/coreweekly.yml (hunk 3 above)
Comment (eternoendless) [reply .github/workflows/coreweekly.yml]: Or weeknumber + year

File: .github/workflows/coreweekly.yml (hunk 4 above)
Comment (eternoendless) [reply .github/workflows/coreweekly.yml]: Or weeknumber + year

File: .github/workflows/coreweekly.yml (hunk 3 above)
Comment (Progi1984) [reply .github/workflows/coreweekly.yml]: Done

File: .github/workflows/coreweekly.yml (hunk 4 above)
Comment (Progi1984) [reply .github/workflows/coreweekly.yml]: Done

File: .github/workflows/coreweekly.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    - name: Install dependencies
+      run: |
+        cd ./core-weekly-generator
+        pip install -r requirements.txt
+    - name: Generate Core Weekly for the current week
+      run: |
+        cd ./core-weekly-generator
+        python core-weekly.py --week ${date +"%U"} > ../prestashop.github.io/news/_posts/core-weekly/${date +"%Y-%m-%d-coreweekly-%U-%Y.md"}
+    - name: Set Week Num
+      run: echo '::set-output name=WEEK_NUM::${date +"%U of %Y"}'
```
Comment (PierreRambaud) [inline .github/workflows/coreweekly.yml]: It's not really the week number, it's confusing with what you have above `--week ` Maybe you should create two variables, one for WEEK_NUM, one for YEAR, and concatenate both for the title?

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matks** [comment]
Bravo !

Next steps, to allow me to stop running the Core Weekly Generator in localhost:
- add a random "good first issue" https://github.com/PrestaShop/core-weekly-generator/issues/76
- highlight PRs labeled "key feature" or "developer feature" https://github.com/PrestaShop/core-weekly-generator/issues/63
- add the "releases" block https://github.com/PrestaShop/core-weekly-generator/issues/31 (PR WIP https://github.com/PrestaShop/core-weekly-generator/pull/62)

**Pull request merged**